### PR TITLE
Updated the tox and travis configs -- BOM-1554

### PR DESCRIPTION
**Pull request by mraarif**
Relevant JIRA issue [here](https://openedx.atlassian.net/browse/BOM-1554).

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -10,66 +10,69 @@ astroid==2.3.3
...
 more-itertools==8.2.0
-newrelic==5.6.0.135
-packaging==20.1
+newrelic==5.12.0.140
+packaging==20.3
 path.py==12.4.0           # via edx-i18n-tools
 path==13.1.0              # via path.py
-pip-tools==4.4.1
+pathlib2==2.3.5
+pip-tools==5.0.0
```
Comment (UsamaSadiq) [inline requirements/dev.txt:44]: You should add constraint on pip-tools. 
`# pip20+ requires for greater version of pip-tools. But installing 19.3.1 in all our containers and VMs.`
`pip-tools==4.5.1`

File: requirements/pip-tools.txt [hunk 2]
```diff
@@ -4,6 +4,9 @@
 #
 #    make upgrade
 #
-click==7.0                # via pip-tools
-pip-tools==4.4.1
+click==7.1.1              # via pip-tools
+pip-tools==5.0.0
```
[the remark this answers is not in the document]
Comment (awais786) [reply requirements/pip-tools.txt:8]:
```
pip20+ requires for greater version of pip-tools. But installing 19.3.1 in all our containers and VMs.
pip-tools==4.5.1
```

File: requirements/dev.txt (hunk 1 above)
Comment (robrap) [reply requirements/dev.txt]: I don’t think the pip-tools constraint is needed any more for libraries. I think it is needed for IDAs. You can remove this constraint.

**Comment by robrap** [comment]
Once @UsamaSadiq approves, this is ready to merge and release.  Thanks.

File: tox.ini [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py35}-django{111},{py35,py36}-django{20,21,22}
+envlist = py{35,38}-django{22,30}
```
Comment (morenol) [inline tox.ini:2]: Django 3.0 dropped support for Python 3.5, so that combination wouldn't work.

File: requirements/base.in [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 -c constraints.txt
 
-Django>=1.11              # Web application framework
+Django                    # Web application framework
```
Comment (morenol) [inline requirements/base.in:5]: I think that the minimum constraint is added in order to report to the dependencies in setup.py which is the minimum supported version of django and this is a library so I think that that should be added in here. But anyway I am going to approve this to see what are the thoughts of @jmbowman about that

**Comment by morenol** [comment]
I forgot to mention that this should be added to the changelog

File: requirements/base.in (hunk 4 above)
Comment (mraarif) [reply requirements/base.in:5]: @jmbowman what do you think?

Comment (jmbowman) [reply requirements/base.in:5]: While the lower boundary doesn't hurt, I'm not sure we really need it either.  I doubt if many people outside the Open edX ecosystem are using this package, and even fewer would be trying to do so with a version of Django so old as to be unsupported upstream.

Comment (mraarif) [reply requirements/base.in:5]: we've already constrained Django in `constraints.txt` to remain <2.3, should we be specifying here?

**Pull request merged**